registers.md

```diff
@@ -393,19 +398,22 @@ Once hashing, when a nonce is found by a chip on the chain, it is sent on the RO
 
 ![](images/nonce.svg)
 ## Write Core Register
-In order to write value to a Core Register, a [Write Register](#write-register) Command shall be done to the [Core Register Control](#core-register-control) Register with the [RD_WR#](#wr_rd) fields = 1.
+In order to write values to a the Core Register, a [Write Register](#write-register) Command shall be done to the [Core Register Control](#core-register-control) Register with the [RD_WR#](#wr_rd) fields = 1.
+
 ## Read Core Register
-In order to read the value of a Core Register, a [Write Register](#write-register) Command shall be done to the [Core Register Control](#core-register-control) Register with the [RD_WR#](#wr_rd) fields = 0.

```

Comment:
no, it is a "Write Register" command send to Register "Core Register Control" with a RD_WR#=1 in order to read a "Core Register".

so please keep it as it was. It is not a copy/paste mistake.